gravityflow/gravityflowincomingwebhook
# Add Expiry Step Setting

This PR/branch provides the ability for incoming step type to leverage the expiration step setting.
![image](https://user-images.githubusercontent.com/4549984/50491520-bb3a5f80-09e0-11e9-9f99-99b216610eee.png)

**Automated Test**
* Run test 0001 which now includes a check for expiration before processing the incoming step

**Manual Test**
* Create a new form + incoming webhook step and define the expiration settings (schedule expiration)
* Submit an entry
* Review the Status Page and note that Expires date/time is listed in the table
* Click into the entry detail page and note that Expires date/time is listed in Workflow info box
* Wait for defined expiration time to expire and note that workflow proceed to appropriate status/step
